ecs-configuration/elasticsearch/ecs-fields-tls.json

```diff
@@ -98,8 +106,16 @@
                 "type": "keyword"
               },
               "ja3s": {
-                "ignore_above": 1024,
-                "type": "keyword"
+                "properties": {
+                  "hash": {
+                    "ignore_above": 8000,
+                    "type": "keyword"
+                  },
+                  "string": {
```

Comment:
Please make this a different field similar to the proposed `ja3.string`. Make this `ja3s_string` instead.